.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!--- Please always add a PR description as if nobody knows anything about the context these changes come from. -->
+<!--- Even if we are all from our internal team, we may not be on the same page. -->
+<!--- Write this PR as you were contributing to a public OSS project, where nobody knows you and you have to earn their trust. -->
+<!--- This will improve our projects in the long run! Thanks. -->
+
+#### Description
+<!--- Describe your changes in detail -->
+
+#### Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
```

Comment:
Isn't decription the same as motivation and context? (it seems reduntant)

Reply:
I wanted to lead the writer in be explicit in both motivation and context, as I see most PRs merely summarize _what_ has been done and they give the _why_ for granted.

Maybe we can collapse both paragraph as one: `#### Description, motivation and context`?

Reply:
maybe we can be more explicit with titles (ie. "List of changes", "Motivation and context")

Reply:
So the suggestion is to keep those separated, but to change `Description` with `List of Changes`?

Reply:
yes